foodsaving/groups/tests/test_tasks.py

```diff
@@ -0,0 +1,53 @@
+from django.core import mail
+from django.utils import timezone
+from django.test import TestCase
+
+from foodsaving.groups.factories import GroupFactory
+from foodsaving.groups.models import GroupMembership
+from foodsaving.groups.tasks import process_inactive_users
+from foodsaving.users.factories import UserFactory
+from config import settings
+
+from datetime import timedelta
+
+
+class TestProcessInactiveUsers (TestCase):
+    def setUp(self):
+        self.activeUser = UserFactory()
```

Comment:
Please use snake_case for variables, as per [PEP8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names). I forgot to add this to the code style checker, will do soon.